### DISCUSSION TITLE
Haptic Feedback

### DISCUSSION THREAD

**Discussion by Wumfg** [gopro/labs#665] [Feature Requests]
I'm working on a project where I need to be able to provide a GoPro user with haptic Feedback for the following events: recording starts, recording stops, camera powering on, and camera shutting down. Each of these events needs to be unique.

 I've played around with using a separate microcontroller to process the speaker chirps into haptic Feedback, but is there a way to use labs in combination with a vibration motor (with or without a microcontroller) to achieve this? Perhaps sending out a signal or power through one of the USB C wires? 

Thanks in advance!

**Comment by dnewman-gpsw** [comment]
Nothing is supported at the moment, and I doubt there are lines on USB C that a free to toggle.
